awx/main/models/execution_environments.py

```diff
@@ -31,6 +32,7 @@ class Meta:
         max_length=1024,
         verbose_name=_('image location'),
         help_text=_("The full image location, including the container registry, image name, and version tag."),
+        validators=[validate_container_image_name],
```

Comment:
:+1: perfect, this will cover the `awx-manage` commands as well.

Reply:
++ this is great